crates/bin/src/args.rs

```diff
@@ -135,7 +142,7 @@ pub struct Args {
     pub(crate) pkg_fmt: Option<PkgFmt>,
 
     /// Override Cargo.toml package manifest pkg-url.
-    #[clap(help_heading = "Overrides", long)]
+    #[clap(help_heading = "Overrides", long, value_name = "URL")]
```

Comment:
```suggestion
    #[clap(help_heading = "Overrides", long, value_name = "URL_TEMPLATE")]
```

Reply:
No objections from my side on this. Makes sense on the basis of what it does. You might go with just TEMPLATE though to keep it short.

Reply:
Agreed, TEMPLATE also look good to me

Reply:
Tweaked wording and value in 74d2fae..19414f8